server/conn_test.go

```diff
@@ -349,12 +349,6 @@ func TestDispatch(t *testing.T) {
 	userData = append(userData, 0x0)
 
 	inputs := []dispatchInput{
-		{
-			com: mysql.ComSleep,
-			in:  nil,
-			err: nil,
```

Comment:
Should be able to assert some errors

Reply:
@xhebox Updated. Please help review. 

Commit: https://github.com/pingcap/tidb/pull/38645/commits/16a5c7d895ca92a554b1563ded33d7e8b1f36345

```go
inputs := []dispatchInput{
        {
            com: mysql.ComSleep,
            in:  nil,
            err: mysql.NewErrf(mysql.ErrUnknown, "command %d not supported now", nil, mysql.ComSleep),
            out: nil,
        },
```